tests/subsys/debug/gdbstub/prj.conf

```diff
@@ -1,5 +1,3 @@
-CONFIG_GDBSTUB=y
```

Comment:
this test is about testing the gdbstub feature in zephyr, so in what cases would this not be needed?

Reply:
> this test is about testing the gdbstub feature in zephyr, so in what cases would this not be needed?

it is not needed for Qemu reference test case.

Reply:
changed to have common config options for gdbstub to be enabled for this test suite